demos/demo-jaas-webapp/src/main/webapp/index.html

```diff
@@ -3,45 +3,50 @@
     <title>JAAS Authentication and Authorization Test</title>
     <meta http-equiv="Pragma" content="no-cache">
     <meta http-equiv="Cache-Control" content="no-cache,no-store">
-    <link rel="stylesheet" type="text/css" href="stylesheet.css"/>
+    <link rel="stylesheet" href="demo.css"/>
+    </style>
 </head>
 <body>
-<a href="https://www.eclipse.org/jetty/"><img src="images/jetty_banner.gif"></A>
-<br/>
-<b><a href="http://localhost:8080/">Demo Home</a></b>
-<hr/>
-<center><span style="color:red; font-variant:small-caps; font-weight:bold">Demo Web Application Only - Do NOT Deploy in Production</span>
-</center>
+    <div class="topnav">
+      <a class="menu" href="http://localhost:8080/">Demo Home</a>
+      <a class="menu" href="https://github.com/eclipse/jetty.project/tree/jetty-10.0.x/demos/demo-jaas-webapp">Source</a>
```

Comment:
Ditto on the manual merge to 11.0.x

Reply:
Yes, It's very very rare that these files change.